### PR TITLE
Converts to ES6 modules and adds eslint

### PR DESCRIPTION
**Pull request by jaredjensen**
This PR:

- Converts scripts to ES6 modules
- Breaks non-UI functionality out into modules
- Adds Docker to avoid CORS errors
- Adds eslint rules to enforce code style
- Adds yarn commands to simplify using the project

### CODE REVIEW

**Review by robert-blue — CHANGES_REQUESTED** [review]
There's a bug in this  PR that causes the TABLE to be drawn one TR at a time and for each TR drawn it pulls down the entirety of the template IDs. Otherwise looks good.

Here's a large set of template IDs that I use:
103370,145826,155285,166103,167332,167345,234120,251305,258614,281655,285053,297137,297139,329691,341946,345393,348878,45017

File: js/script.js [hunk 1]
```diff
@@ -1,227 +1,243 @@
...
+      m.collectionName = lowest.collection_name;
+      m.schemaName = lowest.assets[0].schema.schema_name;
+      m.assetName = lowest.assets[0].name;
+
+      m.historyLink = `https://wax.atomichub.io/market/history?collection_name=${m.collectionName}&data:text.name=${m.assetName}&order=desc&schema_name=${m.schemaName}&sort=updated&symbol=WAX`;
+      m.listingsLink = `https://wax.atomichub.io/market?collection_name=${m.collectionName}&data:text.name=${m.assetName}&order=asc&schema_name=${m.schemaName}&sort=price&symbol=WAX`;
+      m.collectionLink = `https://wax.atomichub.io/explorer/collection/${m.collectionName}`;
+      m.templateLink = `https://wax.atomichub.io/explorer/template/${m.collectionName}/${templateId}`;
+      m.inventoryLink = `https://wax.atomichub.io/profile/${wallet}?collection_name=${m.collectionName}&match=${m.assetName}&order=desc&sort=transferred`;
+    }
```
[the remark this answers is not in the document]
Comment (robert-blue) [reply js/script.js:85]: We might consider including the atomichub npm package which has the return types already mapped out. Might not be worth the bloat though.

File: js/script.js [hunk 2]
```diff
@@ -1,227 +1,243 @@
...
-    while (response.status == 429) {
-      refreshStatusElem.innerText = "AtomicHub rate limit reached. Pausing updates."
-      await sleep(5*1000)
-      
-      response = await(fetch(url))
+    while (response.status === 429) {
+      refreshStatusElem.innerText = 'AtomicHub rate limit reached. Pausing updates.';
+      await util.sleep(5 * 1000);
+
+      response = await (fetch(url));
```
[the remark this answers is not in the document]
Comment (jaredjensen) [reply js/script.js]: I removed the extra parenthesis.  I'm a little surprised eslint doesn't remove them.

File: js/script.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (jaredjensen) [reply js/script.js:85]: I'd probably add that package the first time we run into a problem that the package would have avoided.

**Review by robert-blue — APPROVED** [review]
Looks good

**Pull request merged**